src/components/FilterPane/FilterPane.styles.js

```diff
@@ -120,3 +120,14 @@ export const StyledMenuItem = styled(MenuItem)`
     font-size: ${theme.typography.defaultFontSize};
   }
 `
+export const ExpandClickableArea = styled('div')`
```

Comment:
This isn't following the `Styled...` naming convention. Is this intentional?
What do you think to something like `StyledCheckboxContainer` or similar?